packages/flutter/lib/src/gestures/converter.dart

```diff
@@ -44,8 +44,10 @@ class PointerEventConverter {
   /// [dart:ui.FlutterView.devicePixelRatio]) is used to convert the incoming data
   /// from physical coordinates to logical pixels. See the discussion at
   /// [PointerEvent] for more details on the [PointerEvent] coordinate space.
-  static Iterable<PointerEvent> expand(Iterable<ui.PointerData> data, double devicePixelRatio) sync* {
-    for (final ui.PointerData datum in data) {
+  static Iterable<PointerEvent> expand(Iterable<ui.PointerData> data, double devicePixelRatio) {
+    return data
+        .where((ui.PointerData datum) => datum.signalKind != ui.PointerSignalKind.unknown)
+        .map((ui.PointerData datum) {
       final Offset position = Offset(datum.physicalX, datum.physicalY) / devicePixelRatio;
```

Comment:
should probably indent everything else by 4

Reply:
Done